Person.rb

```diff
@@ -0,0 +1,48 @@
+class Person
+  att_accessor :id, :name, :age
+  def initialize(name='Unknown', age, parent_permission=false)
+    self.age = age
+    self.name = name
+    @parent_permission = parent_permission
+  end
+
+  def setId=(id)
+    :id = id
+  end
+
+  def getId
+    return :id
+  end
+
+  def setName=(name)
+    @name = name
+  end
+
+  def getName
+    return @name
+  end
+
+  def getAge=(age)
+    @age = age
+  end
+
+  def setAge
+    return @age
+  end
```

Comment:
- [ ] since you have  `id` ,`name`, `age` as attr_accessor there is no need for the getter methods and setter methods, kindly remove